packages/ng/form-field/form-field.component.ts

```diff
@@ -165,7 +168,7 @@ export class FormFieldComponent implements OnChanges, OnDestroy, DoCheck {
 		if (this.#control) {
 			// invalid management
 			const previousInvalid = this.invalid;
-			this.invalid = this.#control.invalid && this.#control.touched;
+			this.invalid = (this.#control.invalid || this.statusControl?.invalid) && this.#control.touched;
```

Comment:
If `control` is `invalid` and `statusControl` is `valid`, the `FormField` will still be invalid and the `statusControl` won't drive the validity of the field. Is this the expected result ?

Reply:
Maybe it means that we should rename it but yeah, because you can have validators on both a parent and its child, and one of them being invalid means that you want to mark child as invalid regardless. That's the use case here.